packages/components/organisms/f-takeawaypay-activation/src/tenants/en-GB.js

```diff
@@ -1,4 +1,26 @@
-export default {
+const messages = {
     locale: 'en-GB',
-    text: 'I am a TakeawaypayActivation Component (GB)'
+
+    messages: {
+        titleActivationAvailable: 'Welcome back. Let\'s activate Takeaway Pay on your account.',
```

Comment:
Tiny one: it’s best practice typographically (as it renders nicer in the browser) to use smart quotation marks, rather than what is usual quotation marks. These also have the added benefit of not needing to be escaped (with \ ) when defining inside single quotes in our translation files.

```suggestion
        titleActivationAvailable: 'Welcome back. Let’s activate Takeaway Pay on your account.',
```

I am happy for this change to go to your next PR

Reply:
Thanks @dandel10n . Sure, will update this in the next PR